### PR TITLE
Attachment removal zone

### PR DESCRIPTION
**Pull request by RebeccaStankus**

### CODE REVIEW

File: Shared/AttachmentRemovalZone/attachmentRemoval.js [hunk 1]
```diff
@@ -0,0 +1,24 @@
...
+//  See the accompanying file LICENSE or http://www.apache.org/licenses/LICENSE-2.0.html
+//
+//  This script acts on a zone to make it remove all avatar attachments when an avatar enters the zone.
+
+(function () {
+    var shared = Script.require('./attachmentZoneShared.js');
+    var _this = this;
+    _this.enterEntity = function (entityID) {
+        shared.getAvatarChildEntities(MyAvatar).forEach(function (entityID) {
+            var childUserData = JSON.stringify(Entities.getEntityProperties(entityID).userData);
```
Comment (thoys) [inline Shared/AttachmentRemovalZone/attachmentRemoval.js]: You don't need to stringify this, it is already a string.

**Comment by RebeccaStankus** [comment]
TEST PLAN
Run "spawnAttachmentRemovalZone.js" and have items nearby that use the "attachmentItemScript.js" such as items from the store in empty. 

--Have your avatar try on one item and then walk up the ramp onto the platform.
--The item should disappear. Items that are not attached should not disappear.
--Repeat with multiple items
--All attached items should disappear upon entering the platform area.

File: DomainContent/AvatarStore/AttachmentRemovalZone/spawnAttachmentRemovalZone.js [hunk 2]
```diff
@@ -0,0 +1,37 @@
...
+    var entityTree = _entityImport.importEntitiesJSON(jsonUrl, {
+        position: getPosition()
+    });
+
+    var laundryBasketParts = _entityImport.createEntitiesFromTree([
+        entityTree
+    ])[0].childEntities;
+
+    Script.scriptEnding.connect(cleanup);
+})();
```
Comment (misslivirose) [inline DomainContent/AvatarStore/AttachmentRemovalZone/spawnAttachmentRemovalZone.js]: EOF newline

Comment (misslivirose) [reply DomainContent/AvatarStore/AttachmentRemovalZone/spawnAttachmentRemovalZone.js]: is this indentation the eslint suggestion? it seems kind of odd.

File: DomainContent/AvatarStore/AttachmentRemovalZone/spawnAttachmentRemovalZone.js [hunk 3]
```diff
@@ -0,0 +1,37 @@
...
+//
+//  Created by Rebecca Stankus on 9/28/17.
+//  Copyright 2017 High Fidelity, Inc.
+//
+//  Distributed under the Apache License, Version 2.0.
+//  See the accompanying file LICENSE or http://www.apache.org/licenses/LICENSE-2.0.html
+//
+//  This script creates a laundry basket to get rid of attachments after trying them on.
+(function () {
+    var _entityImport = Script.require('https://hifi-content.s3.amazonaws.com/rebecca/entityImport.js');
```
Comment (misslivirose) [inline DomainContent/AvatarStore/AttachmentRemovalZone/spawnAttachmentRemovalZone.js]: These variables are constants and should be named as such following the standards

**Pull request merged**